modules/_labs/header/react/lib/parts/SearchBar.tsx

```diff
@@ -195,61 +213,63 @@ const CloseButton = styled(IconButton)<
 
   return {
     position: `absolute`,
-    top: spacing.zero,
-    right: spacing.zero,
-    bottom: spacing.zero,
+    top: 0,
+    bottom: 0,
+    right: 0,
```

Comment:
At this point, I think `spacing.zero`, `spacingNumbers.zero`, and `0` are all interchangeable, I was trying to figure out a rationale for all this but ended up making it more inconsistent. I'm okay leaving unless someone can tell me the official reason why we have the `zero`.

Even if we were to change our spacing system, I would imagine that `zero` always refers to `0`?